client/utils.go

```diff
@@ -152,17 +153,24 @@ func authortext(AuthorText sql.NullString, authors []database.Author) string {
 	var sb strings.Builder
 	sb.WriteString("Skriven av ")
 
-	sb.WriteString(authorsName(authors[0]))
+	sb.WriteString(string(authorURL(authorsName(authors[0]), authors[0].KthID)))
 	if len(authors) == 1 {
-		return sb.String()
+		return template.HTML(sb.String())
 	}
 
 	for i := 1; i < len(authors)-1; i++ {
-		sb.WriteString(fmt.Sprintf(", %v", authorsName(authors[i])))
+		sb.WriteString(fmt.Sprintf(", %v",
+			authorURL(authorsName(authors[i]), authors[i].KthID)))
 	}
 
-	sb.WriteString(fmt.Sprintf(" och %v", authorsName(authors[len(authors)-1])))
-	return sb.String()
+	sb.WriteString(fmt.Sprintf(" och %v",
+		authorURL(authorsName(authors[len(authors)-1]), authors[len(authors)-1].KthID)))
+	return template.HTML(sb.String())
+}
+
+func authorURL(name string, kthID string) template.HTML {
+	url := fmt.Sprintf(`<a href="/redaqtionen/%v">%v</a>`, kthID, name)
```

Comment:
Same here as above. You should probably move some logic into the template rather than `Sprintf`ing html here. Or this function could call a very small template.